config/app.default.php

```diff
@@ -286,19 +286,35 @@
      * Configures logging options
      */
     'Log' => [
+        '404' => [
+            'className' => 'Cake\Log\Engine\FileLog',
+            'path' => LOGS,
+            'file' => '404',
+            'url' => env('LOG_NOT_FOUND_URL', null),
+            'scopes' => ['404'],
```

Comment:
But could we? We can rename it to NotFound? Otherwise I can remove it. However it helped me quite a lot spotting missing files VS not polluting my other log files.